src/ShaderGen.Tests/TestAssets/ComputeShaderParameters.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System.Numerics;
+using System.Runtime.InteropServices;
+
+namespace TestShaders
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public unsafe struct ComputeShaderParameters
+    {
+        public /*readonly*/ Matrix4x4 P1Matrix;
+        public /*readonly*/ Matrix4x4 P2Matrix;
+
+        public /*readonly*/ Vector4 P1Vector4;
+        public /*readonly*/ Vector4 P2Vector4;
+        public /*readonly*/ Vector4 P3Vector4;
+        public /*readonly*/ Vector4 OutVector4;
+
+        public /*readonly*/ Vector3 P1Vector3;
+        public /*readonly*/ float P1Float;
+        public /*readonly*/ Vector3 P2Vector3;
+        public /*readonly*/ float P2Float;
+        public /*readonly*/ Vector3 P3Vector3;
+        public /*readonly*/ float P3Float;
+        public /*readonly*/ Vector3 OutVector3;
+        public /*readonly*/ float OutFloat;
+
+        public /*readonly*/ Vector2 P1Vector2;
+        public /*readonly*/ Vector2 P2Vector2;
+        public /*readonly*/ Vector2 P3Vector2;
+        public /*readonly*/ Vector2 OutVector2;
+
+        /*
```

Comment:
Can this be removed?

Reply:
Sorry, this was in one of the commits I rolled back.  I had it in there whilst it was WIP to remind me of some of the things that ShaderGen doesn't currently do, or do well - as I've raised issues for those it should have been taken out.